### ISSUE TITLE
stomach metabolises injected reagents

### ISSUE THREAD

**Issue by deltanedas** [space-wizards/space-station-14#20923]
## Description
it shouldnt do that since its not going into the stomach duh

it might need adding a new solution container specifically for reagents that go into the stomach
mouth (none) -> stomach (food+drink) -> temp bloodstream???? (?) -> bloodstream (poison+narcotic+medicine)

**Reproduction**
1. inject yourself with water
2. instead of bubbles in your blood being bad you just stop being thirsty

**Comment by notquitehadouken** [comment]
air embolism meta

**Comment by deltanedas** [comment]
yes when smoke worked we did that a lot just smoke some soda and it works

**Comment by notquitehadouken** [comment]
needle too long, goes directly through flesh into stomach

**Comment by ThatGuyUSA** [comment]
think this was changed in #42172, and shouldn't be applicable anymore

**Comment by Princess-Cheeseballs** [comment]
Yep was changed.
